### ISSUE TITLE
activiti 5 processes stopped working after upgrade

### ISSUE THREAD

**Issue by masrawi** [Activiti/Activiti#782]
after upgrading to version 6 I have a Nullpointer in all my processes :-(  `JobEntityManagerImpl` line 322.

after debugging it seems that the `jobEntity.getJobHandlerType()` returns `timer-intermediate-transition`
which is not in the map which contains only:

```
timer-start-event               
trigger-timer                   
activate-processdefinition      
suspend-processdefinition       
event                           
async-continuation              
```

**Comment by tijsrademakers** [comment]
Thanks for reporting this issue, we are looking for this kind of feedback before we can release a first official version of the Activiti 6 Engine. We'll look into this issue and make sure it gets covered.

**Comment by jbarrez** [comment]
@masrawi : could it be that this is also caused by not running with v5 compatibility enabled?

**Comment by masrawi** [comment]
yes it could be. I did not set any flag other than the default

**Comment by Salaboy** [comment]
Your issue is being closed now due to inactivity for a long period.  We want to keep the project repository healthy and for that reason, we will close inactive issues. If you are interested in reviewing this issue with us so we can find a way to close it we will appreciate you opening a new issue pointing out the correct branch where the issue is present. 
We appreciate your support and understanding.
